applications/social_recovery_wallet.md

```diff
@@ -0,0 +1,200 @@
+
+## Team :busts_in_silhouette:
+
+### Team Members
+- Nikolaus Heger - Flutter & Blockchain Developer https://github.com/n13
+- Gerardo Guijarro  - Flutter Developer https://github.com/gguijarro-c-chwy
+- Jason Doh - UX Design
+- Other team members as needed
+
+We have already been learning Substrate and we are partnering with the [Hashed Network](https://github.com/hashed-io). Many of our users are already planned to migrate over. 
+
+### Contact
+- **Contact Name:** Nikolaus Heger
+- **Contact Email:** nheger@gmail.com
+- **Website:** https://hypha.earth
+
+### Legal Structure
+- **Registered Address:** 30 N. Gould St, Ste R, Sheridan, Wyoming
+- **Registered Legal Entity:** Hypha DAO LLC
+
+### Team's experience
+
+We have deployed and implemented the Seeds Light Wallet. Nikolaus Heger and Gerardo Guijarro specced, implemented, and deployed social recovery for the Seeds Light Wallet. We also implemented the smart contracts. Smart contracts were deployed on Telos, an EOSIO fork. Hypha is a Hashed Network launch partner.  
+
+### Team Code Repos
+
+Hypha develops two products, the Hypha DAO, and the SEEDS regenerative currency, DAO, and ecosystem. 
+
+- https://github.com/JoinSEEDS
+- https://github.com/JoinSEEDS/seeds_light_wallet
+- https://github.com/JoinSEEDS/seeds-smart-contracts
+- https://github.com/JoinSEEDS/seeds-smart-contracts/blob/master/include/seeds.guardians.hpp
+
+- https://github.com/hypha-dao
+- https://github.com/hypha-dao/dao-contracts
+
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/nikolausheger
+- https://www.linkedin.com/in/gguij002/
+- https://www.linkedin.com/in/jasonkihyun/
+
+## Development Status :open_book:
+
+We have started research of how to integrate Polkadot JS into the flutter app. 
+
+https://github.com/JoinSEEDS/seeds_light_wallet/tree/feature/polkadot1
+
+We have deployed a full implementation in the Seeds Light Wallet
+
+We created a video showing off how the feature works on Telos - it's going to be a bit different due to the way Polkadot recovery pallet works, but 90% of the design and specifications are going to be the same (wait on recover, prevent hacks, set up guardians). It's less than 5 minutes, enjoy! 
+
+https://www.youtube.com/watch?v=Vpb8IbHvcNU
+
+## Development Roadmap :nut_and_bolt:
+### Overview
+- **Total Estimated Duration:** 3 months
+- **Full-Time Equivalent (FTE):**  2.5 FTE 
+- **Total Costs:** 50,000 USD
+
+#### Languages
+- All deliverables will be written in Flutter/Dart. 
+
+## Milestones
+For each milestone, we will deliver documentation and a video.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | MIT |
+| 0b. | Documentation | We will provide **inline documentation** of the code. |
+| 0c. | Video | We will record and publish a video explainer and demonstration of all features. |
```

Comment:
I understand, that at first sight it might make sense to declare these deliveries only once while avoiding redundancy (DRY), instead of copying them into each milestone. However, for various reasons, we'd prefer to have them in each milestone.
Please also note that [Deliverables 0a-0d are mandatory for all milestones](https://github.com/w3f/Grants-Program/blob/8556945d606d3f64c3a8a04566cc6d486a7dd680/applications/application-template.md?plain=1#L116), unless a reason is provided why this shouldn't be followed. Of course, you're free to deliver _0e. Video_ in addition to these.

Reply:
Thank you for this clarification - I missed this. 

0d is a dockerfile? Is this also required for mobile apps? Our app will run on Android and iOS, I was thinking delivery would be through Apple and Google app stores, the respective testing tracks (open testing / Testflight)

I imagine maybe it is possible to wrap an android simulator into a docker file... 

I am more than happy to rectify this and include these into all milestones. Thank you.

For reference:
```
| 0a. | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
| 0b. | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
| 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
| 0e. | Article | We will publish an **article**/workshop that explains [...] (what was done/achieved as part of the grant). (Content, language and medium should reflect your target audience described above.)

```

Reply:
Ok I have made the requested changes.

Reply:
As for docker vs app delivery - I did a brief search and it is possible to launch Flutter unit tests from a docker container, but the docker container can't start an app in the native simulator - so it's really not all that much fun. 

We would be more than happy to deliver a dockerized flutter that runs the unit tests, if that is desired.

Reply:
Thanks for your research. I think the main reason we require docker is that it allows us to run the whole project in an easy and integrative way. Since this doesn't seem to be possible for mobile apps, I'm fine with skipping it, as long as you can provide an easy way to run the unit tests. 👍